_data/people/meevans1.yml

```diff
@@ -0,0 +1,68 @@
+name: <Meirin Oan Evans>
```

Comment:
No `<>` brackets, please. :)

Reply:
Resolved in commit https://github.com/iris-hep/iris-hep.github.io-source/pull/534/commits/420ebd062d598e9fc399a5e99d653b2da082e198